packages/lib/src/select/Select.tsx

```diff
@@ -52,6 +52,7 @@ const DxcSelect = forwardRef<RefType, SelectPropsType>(
       margin,
       size = "medium",
       tabIndex = 0,
+      ariaLabel = "Select input",
```

Comment:
As with the slider, I suggest removing the input word and having the same component name.